.kitchen.yml

```diff
@@ -1,35 +1,40 @@
 ---
+<% chef_versions = %w( 13 14 ) %>
+<% platforms = %w( centos-6 centos-7 ) %>
+
 driver:
   name: vagrant
   linked_clone: true
-  require_chef_omnibus: 12
 
 provisioner:
   name: chef_zero
+  log_level: <%= ENV['TK_CHEF_LOG_LEVEL'] || 'info' %>
 
 verifier:
   name: inspec
   inspec_tests:
     - test/inspec
 
 platforms:
-  - name: centos-6.8
-  - name: centos-7.3
-
-suites:
-  - name: config_file
-    run_list:
-      - recipe[knotx::_install]
+<% platforms.each do |p| %>
+<%   chef_versions.each do |chef_version| %>
+  - name: <%= p %>-chef-<%= chef_version %>
+    driver_config:
+      box: bento/<%= p %>
+      require_chef_omnibus: <%= chef_version %>
     attributes:
       knotx:
-        debug_enabled: true
-  - name: config_git
+        url: 'https://oss.sonatype.org/content/groups/public/io/knotx/knotx-stack-manager/1.3.0-SNAPSHOT/knotx-stack-manager-1.3.0-20180417.122358-11.zip'
+      java:
+        jdk:
+          '8':
+            x86_64:
+              url: <%= ENV['JAVA_JDK_8_X86_64_URL'] || 'http://download.oracle.com/otn-pub/java/jdk/8u162-b12/0da788060d494f5095bf8624735fa2f1/jdk-8u162-linux-x64.tar.gz' %>
```

Comment:
I'm getting 404 on this. It's possible that Oracle once again changed URLs.

Reply:
JDK link has been updated to the newest version. To download that file you have to provide a cookie `curl -L --cookie oraclelicense=accept-securebackup-cookie http://download.oracle.com/otn-pub/java/jdk/8u172-b11/a58eab1ec242421181065cdc37240b08/jdk-8u172-linux-x64.tar.gz -o /dev/null -`